scripts/paypalBillingSetup.js

```diff
@@ -73,7 +73,7 @@ switch(OP) {
     });
     break;
   case "create":
-    paypal.billingPlan.create(blocks["google_6mo"].definition, function(err,plan){
+    paypal.billingPlan.create(blocks["focusme_6mo"].definition, function(err,plan){
```

Comment:
As far as I know, Google employees can still take advantage of that discount (@lemoness, confirm?), so we're not removing the google_6mo plan.

This particular bit, however, doesn't affect existing plans. It's a bit of script to make a call to the PayPal API to create a new plan. I could've just made the change locally before running the script, but wanted it in the commit for people to see in case I am mistaken about how it works!